src/name_bazaar/ui/spec.cljs

```diff
@@ -146,3 +150,6 @@
                                                 ::offerings-main-search-drawer]
                                        :opt-un [::public-resolvers]
                                        :opts [:offerings/total-count])))
+
+(defn generate-registrar-bid-salt []
+  (g/generate (s/gen :registrar/bid-salt)))
```

Comment:
There's function district0x.shared.utils/rand-str you can just use that, so less code needs to be included in app.
Besides, this function shouldn't be in `name-bazaar.ui.spec` at all